js/core/dom_component.js

```diff
@@ -12,7 +12,7 @@ import { grep, noop } from './utils/common';
 import { inArray } from './utils/array';
 import { isString, isDefined } from './utils/type';
 import { hasWindow } from '../core/utils/window';
-import { resize as resizeEvent, visibility as visibilityEvents } from '../events/core/events_engine';
+import { resize as resizeEvent, visibility as visibilityEvents } from '../events/index';
```

Comment:
Should we remove `/index` from this line because of an `index` file will be exported by default from folder?

Reply:
We already have `events.js` file